### DISCUSSION TITLE
Upgrade from decrecated reporting

### DISCUSSION THREAD

**Discussion by jd1900** [Koenkk/zigbee2mqtt#6596] [General]
How may I upgrade from deprecated reporting deviced to the new way of working?
I've seen that now devices in the "reporting" tab have some attributes defined to be reported. But not in the previously added devices.

How do I "upgrade" device to work that way?

**Comment by Koenkk** [comment]
See second point of https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.17.1

**Reply by jd1900** [reply]
Thank you.
Is it possible to make it automatic?

I've tried with "reconfigure" but it doesn't add all the default reporting. And with dozens of devices it can be quite slow and in some devices I don't know what to setup reporting for.

**Reply by Koenkk** [reply]
It's not possible to do this automatic. But the reportings that are setup with reconfigure are already restored after this. You only have to do it for binds (https://www.zigbee2mqtt.io/information/binding.html)
